zaza/charm_lifecycle/deploy.py

```diff
@@ -54,9 +54,7 @@ def get_charm_config_context():
     test_config = utils.get_charm_config()
     ctxt = {
         'charm_name': test_config['charm_name'],
-        'charm_location': '{}/../../../{}'
-                          .format(bundle_dir_abspath,
-                                  test_config['charm_name']),
+        'charm_location': '{}/../../.'.format(bundle_dir_abspath),
```

Comment:
The `../../.` looks a bit strange.  Surely `../..` is the same path?

Also, if `bundle_dir_abspath` is an abspath, why not use `os.path.abspath` to make an abs path?

```python
In [4]: os.path.abspath("{}/../..".format("/some/interesting/path/here"))       
Out[4]: '/some/interesting'

In [5]: os.path.abspath("{}/../../.".format("/some/interesting/path/here"))     
Out[5]: '/some/interesting'
```

Reply:
Thanks for the feedback! I've addressed these in the next revision.